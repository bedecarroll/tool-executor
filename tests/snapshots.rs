use assert_cmd::Command;
use insta::assert_snapshot;

#[test]
fn greet_command_snapshot() -> color_eyre::Result<()> {
    let mut cmd = Command::cargo_bin("tx")?;
    let assert = cmd.args(["greet", "--name", "snapshot"]).assert().success();
    let stdout = String::from_utf8(assert.get_output().stdout.clone())?;
    assert_snapshot!("greet_command_snapshot", stdout);
    Ok(())
}
