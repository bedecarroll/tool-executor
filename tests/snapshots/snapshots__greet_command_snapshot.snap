---
source: tests/snapshots.rs
assertion_line: 9
expression: stdout
---
Hello, snapshot!
